### PR TITLE
Update Readme: CR Methods citation, and Slack badge

### PR DESCRIPTION
**Pull request by rgutzen**
- Replacing the arxiv citation with the [CR Methods citation](https://doi.org/10.1016/j.crmeth.2023.100681).
- Adding a badge to join the Cobrawap Slack

### CODE REVIEW

**Review by cosimolupo — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -12,6 +12,11 @@ Collaborative Brain Wave Analysis Pipeline (Cobrawap)
   :alt: DOI Latest Release
   :align: left
 
+.. image:: https://img.shields.io/badge/slack-join-pink.svg
+   :target: https://join.slack.com/t/cobrawapworkinggroup/shared_invite/zt-1t8fjv447-64MrlHywww97LRC1ZtW0DA
+   :alt: Join our Slack Community
+   :align: left
```
Comment (gulpgiulia) [inline README.rst:18]:
```suggestion
   :align: right
```

Comment (gulpgiulia) [reply README.rst:18]: not sure if "align left" would work when displayed on screen, because there are three banners in the same line now

Comment (rgutzen) [reply README.rst:18]: The alignment to the left for all the elements has the effect that they are listed one after the other from left to right. You can see the rendered version of the README on the fork branch.
However, it seems that the displayed order (doi, slack, docs) is not the same as in the rst code (docs, doi, slack). I'm not sure how the display order is determined then.

Comment (gulpgiulia) [reply README.rst:18]: I can't see your fork branch (I see there are 14 forks, but just mine is visible to me); my preview of the .rst code here in the github review tool shows the "slack" banner in a new line, below the others

**Comment by mdenker** [comment]
Wait with merge until after hotfix.

**Comment by mdenker** [comment]
Merge as soon as version number is increased, e.g., after https://github.com/NeuralEnsemble/cobrawap/pull/62 https://github.com/NeuralEnsemble/cobrawap/issues/63

**Review by mdenker — APPROVED** [review]

**Pull request merged**